src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/PresentationBuildTasks.csproj

```
@@ -308,14 +298,16 @@
   </ItemGroup>
```

```suggestion
    <Reference Include="System.CodeDom" />
  </ItemGroup>
```

Not sure if you need this or not?

---

Not needed on netfx